src/resources/js/commerce/notice.js

```diff
@@ -0,0 +1,137 @@
+/* global tribe, jQuery */
+/**
+ * Makes sure we have all the required levels on the Tribe Object
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets = tribe.tickets || {};
+
+/**
+ * Configures ET Tickets Commerce Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets.commerce = tribe.tickets.commerce || {};
+
+/**
+ * Configures ET Tickets Commerce Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {Object}
+ */
+tribe.tickets.commerce.notice = tribe.tickets.commerce.notice || {};
+
+/**
+ * Initializes in a Strict env the code that manages the plugin tickets commerce.
+ *
+ * @since TBD
+ *
+ * @param  {Object} $   jQuery
+ * @param  {Object} obj tribe.tickets.commerce
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	const $document = $( document );
+
+	/*
+	 * Tickets Commerce Selectors.
+	 *
+	 * @since 5.1.9
+	 */
+	obj.selectors = {
+		hiddenElement: '.tribe-common-a11y-hidden',
+		item: '.tribe-tickets__commerce-checkout__notice',
+		content: '.tribe-tickets__commerce-checkout__notice__content',
```

Comment:
Let's work to make these BEM friendly.

Would be `tribe-tickets__commerce-checkout-notice` and `tribe-tickets__commerce-checkout-notice-content`